timescale_vector/client.py

```diff
@@ -433,13 +434,29 @@ def build_query(self, params: List) -> Tuple[str, List]:
                     params.append(value)
                     continue
 
+                # if array containment
+                if operator == "@>" and isinstance(value, list):
+                    field_cast = ''
```

Comment:
how are string values handled? Should strings be tested for below? or should the default cast be to text[]?

Reply:
we should probably add a test for string-typed value array

Reply:
good catch.
actually, i just realized that I can make the code much more flexible and simpler...

Reply:
latest push allows for PredicateValues that are tuples or lists containing values of mixed types